tests/10-issue.js

```diff
@@ -356,12 +421,15 @@ describe('Issuers - BitstringStatusList',
             'bit in the bitstring.',
           async function() {
             this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20bitstring%20MUST,Bitstring%20Encoding.';
-            const {slc: {credentialSubject}} = await getSlc({issuedVc});
-            const {encodedList} = credentialSubject;
-            const decoded = await sl.decodeList({encodedList});
-            decoded.bitstring.bits[0].should.be.equal(0);
-            decoded.bitstring.bits[
-              decoded.bitstring.bits.length - 1].should.be.equal(0);
+            for(statusListCredential in statusListCredentials) {
```

Comment:
the name here kind of concerns me. if statusListCredentials is an array then `in` should give the index, if `statusListCredentials` is an object then `in` will give a property name or a key. p.s. if statusListCredentials is an Array you can use `of` and directly iterate the values in the array.